Xamarin.Forms.Platform.Tizen/ExportRendererAttribute.cs

```diff
@@ -1,12 +1,16 @@
 using System;
+using Xamarin.Forms.Core;
+using Information = Tizen.System.Information;
 
 namespace Xamarin.Forms.Platform.Tizen
 {
 	[AttributeUsage(AttributeTargets.Assembly, AllowMultiple = true)]
-	public sealed class ExportRendererAttribute : HandlerAttribute
+	public sealed class ExportRendererAttribute : BaseExportRendererAttribute
 	{
 		public ExportRendererAttribute(Type handler, Type target) : base(handler, target)
 		{
+			if(Information.TryGetValue("tizen.org/feature/platform.version", out string OSVersion))
+				MajorVersion = Convert.ToInt32(OSVersion.Split('.')[0]);
```

Comment:
Here, I assumed version parts would be separated by periods like on other platforms. Not sure if that's the case on Tizen.

Reply:
Yes, platform version is separated by periods something like 4.0.0 on Tizen. However, as I mentioned above, API version is much clear. The API Version in Tizen is almost the same as the API Level in Android. The current/latest version of Tizen API is `4`.

`MajorVersion = Tizen.Common.DotnetUtil.TizenAPIVersion`